tensorflow/core/ops/compat/ops_history.v1.pbtxt

```diff
@@ -59828,6 +59828,34 @@ op {
   }
   is_stateful: true
 }
+op {
```

Comment:
you should not need to make this change if you make the op experimental and hidden in the base API